examples/cancer.py

```diff
@@ -2,7 +2,7 @@
 from sklearn.datasets import load_breast_cancer
 import numpy as np
 import theano
-from sklearn.cross_validation import train_test_split
+from sklearn.model_selection import train_test_split
```

Comment:
I like moving towards newer API here as this was introduced about version 0.18 of scikit-learn. It would be also good to reflect this in setup file by adding `scikit-learn>=0.18` (this is not recent version so it's ok not to support older versions of scikit-learn).